src/zimdump.cpp

```diff
@@ -444,9 +444,12 @@ void ZimDumper::dumpFiles(const std::string& directory, bool symlinkdump)
         std::string redirectUrl = redirectArticle.getUrl();
         if (symlinkdump == false && redirectArticle.getMimeType() == "text/html")
         {
+            auto encodedurl = urlEncode(redirectUrl, true);
             std::ostringstream ss;
             ss << ("<meta http-equiv=\"refresh\" content=\"0\"; url=\"" +
-                    urlEncode(redirectUrl, true) + "\" />");
+                    encodedurl + "\" />");
+            ss << ("<script type=\"text/javascript\"> window.location.href = \"" +
```

Comment:
@MiguelRocha Why we need to do so? Why a standard HTML redirect does not work?

Reply:
@kelson42 firefox is not behaving correctly with the meta redirect tag. I'll test with other browsers but my bet is that not all browsers implement this specific tag

Reply:
@MiguelRocha I think I had tested with Chromium as well and it was not better. I hardly believe we face a bug in Firefox here.

Reply:
@MiguelRocha I had a look and the problem is just that your HTML is not valid. It should be here:
```
<!DOCTYPE html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta http-equiv="refresh" content="0;url=Code_de_d%C3%A9ontologie_m%C3%A9dicale" />
<head>
<body></body>
</html>
```

Reply:
Thanks. I did it and it works